### ISSUE TITLE
fd-link not sufficient touch target size

### ISSUE THREAD

**Issue by esporysz** [SAP/fundamental-ngx#12779]
#### Is this a bug, enhancement, or feature request?

bug

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)

angular: 18.2.4
fundamental: 0.53.1-rc.23

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;

1. According to [documentation](https://www.w3.org/TR/WCAG22/#target-size-minimum) touch target size should be at least 24px by 24 px. Currently fd-link is smaller

**Comment by droshev** [comment]
@nikolay-kolarov What is the recommendation for us to do?

**Comment by droshev** [comment]
``` And in the case when the link is inside table cell it is long enough and there is a free space around the link so touch target size is sufficient.```
